_posts/en/newsletters/2019-07-24-newsletter.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Bitcoin Optech Newsletter #56'
+permalink: /en/newsletters/2019/07/24/
+name: 2019-07-24-newsletter
+slug: 2019-07-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes progress on signet and an idea for just-in-time
+routing on LN.  Also included are our regular sections about bech32
+sending support and notable changes to popular Bitcoin infrastructure
+projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.1 release candidates:** this upcoming
+  maintenance release fixes several bugs that affected certain RPC
+  commands and caused unnecessarily high CPU use in certain cases.
```

Comment:
Perhaps s/certain/some/ to not use "certain" twice in the same sentence?

Reply:
Changed first instance to "some"